typescript/packages/subsurface-viewer/src/layers/utils/layerTools.ts

```diff
@@ -20,7 +20,7 @@ import type DrawingLayer from "../drawing/drawingLayer";
 export type Position3D = [number, number, number];
 
 // Return a color given a number in the [0,1] range.
-export type colorMapFunctionType = (x: number) => [number, number, number];
+export type ColorMapFunctionType = (x: number) => [number, number, number];
```

Comment:
Can we export also the old `colorMapFunctionType` aliased to the new type, and mark it deprecated? So that we don't make it a breaking change.